fee_calculator/apps/api/filters.py

```diff
@@ -57,8 +57,8 @@ def filter(self, qs, value):
         if self.distinct:
             qs = qs.distinct()
         qs = self.get_method(qs)(
-            Q(**{'%s__isnull' % self.field_name: True}) |
-            Q(**{'%s__%s' % (self.field_name, lookup): value})
+            Q(**{'%s__isnull' % self.name: True}) |
```

Comment:
@ibrechin could adding this error back in be causing a complete internal server error on staging https://laa-fee-calculator-staging.apps.cloud-platform-live-0.k8s.integration.dsd.io/